numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1018,7 +1018,7 @@ def add_newdoc(place, name, doc):
     """
     Compute the Heaviside step function.
 
-    The Heaviside step function is defined as::
+    The Heaviside step function [1]_ is defined as::
```

Comment:
The footnote in the [rendered doc](https://output.circle-artifacts.com/output/job/198862a1-ea91-445f-bce3-e200e584b975/artifacts/0/doc/build/html/reference/generated/numpy.heaviside.html#id2) seems broken.